nca/NetworkConfig/ResourcesHandler.py

```diff
@@ -45,6 +57,37 @@ def set_global_peer_container(self, global_ns_list, global_pod_list, global_reso
         self._set_config_peer_container(global_ns_list, global_pod_list, global_resource_list,
                                         'global', True, global_resources_parser)
 
+    def analyze_livesim(self, peer_container, policy_finder):
+        """
+        Analyze the pre-parsing of the topology and finds what needs
+        to be added.
+        :param PeerContainer peer_container: Contains the peers found in pre-parsing
+        :param PolicyFinder policy_finder: Contains the policies found in pre-parsing
+        :return: [strings]: configuration_addons: the paths of the yamls to be added.
+        """
+        configuration_addons = []
+        path = os.path.dirname(__file__)
+        # find kube-dns reference
+        if policy_finder.missing_pods_with_labels.get('name') == 'kube-system':
+            configuration_addons.append(os.path.join(path, LiveSimPaths.DnsCfgPath))
+
+        # find ingress controller pods
+        if policy_finder.found_ingress_control_policy:
+            ingress_controller_found = False
+            ingress_controllers = ['ingress-nginx', 'ingress-gce', 'app-ingress']
```

Comment:
The ingress policy parser only supports `ingress-nginx` currently, so if you find only `ingress-gce` and skip adding the 
`ingress-nginx`  service, the parser will issue a warning that it is missing and will ignore the ingress policy.

Reply:
Done. 
Currently (after fix), only one place holds the names.